libvisual-plugins/plugins/actor/gstreamer/actor_gstreamer.c

```diff
@@ -161,7 +180,9 @@ static void act_gstreamer_cleanup (VisPluginData *plugin)
         gst_element_set_state (priv->pipeline, GST_STATE_NULL);
         gst_object_unref (priv->pipeline);
 
-        gst_buffer_unref (priv->buffer);
+        if (priv->buffer) {
+            gst_buffer_unref (priv->buffer);
+        }
 
 #if GLIB_VERSION_CUR_STABLE >= GLIB_VERSION_2_32
```

Comment:
This Glib version check should no longer be needed. I looked through GStreamer's repository and it looks like GStreamer 1.0 already requires 2.32 ([link](https://gitlab.freedesktop.org/gstreamer/gstreamer/-/blob/1.0/configure.ac#L567)) to build.

Reply:
@kaixiong good idea, done.